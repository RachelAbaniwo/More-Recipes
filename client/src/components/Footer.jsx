
import React from 'react';
/**
 * Display footer component
 * @param {null} null
 *
 * @returns {object} jsx for footer
 *
 */
const Footer = () => ((
  <footer
    className="fixed-bottom"
    style={{
      backgroundColor: 'rgba(73, 67, 67, 0.9)',
      maxWidth: '100%',
      height: 30
    }}
  >
    <p
      className="text-center"
      style={{
          marginBottom: 5,
          paddingTop: 5,
          color: '#fff',
          fontSize: 12
        }}
    >
          Â© 2017 More-Recipes
    </p>
  </footer>
));

export default Footer;
