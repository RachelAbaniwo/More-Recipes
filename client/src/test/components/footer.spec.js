import React from 'react';
import renderer from 'react-test-renderer';

import Footer from './../../components/Footer.jsx';

describe('The footer component', () => {
  it('Should render without crashing', () => {
    const tree = renderer.create(<Footer/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
