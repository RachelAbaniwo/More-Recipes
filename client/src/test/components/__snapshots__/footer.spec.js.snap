// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The footer component Should render without crashing 1`] = `
<footer
  className="fixed-bottom"
  style={
    Object {
      "backgroundColor": "rgba(73, 67, 67, 0.9)",
      "height": 30,
      "maxWidth": "100%",
    }
  }
>
  <p
    className="text-center"
    style={
      Object {
        "color": "#fff",
        "fontSize": 12,
        "marginBottom": 5,
        "paddingTop": 5,
      }
    }
  >
    Â© 2017 More-Recipes
  </p>
</footer>
`;
